addons/website/static/src/builder/plugins/options/avatars_option.xml

```diff
@@ -0,0 +1,135 @@
+        />
+    </BuilderRow>
+    <BuilderRow label.translate="Order">
+        <BuilderSelect>
+            <BuilderSelectItem classAction="''">Last on top</BuilderSelectItem>
+            <BuilderSelectItem classAction="'o_first_on_top'">First on top</BuilderSelectItem>
+        </BuilderSelect>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Alignment">
+        <BuilderButtonGroup>
+            <BuilderButton
+                className="'fa fa-fw fa-align-left'"
+                title.translate="Left"
+                classAction="''"
+            />
+            <BuilderButton
+                className="'fa fa-fw fa-align-center'"
+                title.translate="Center"
+                classAction="'o_avatars_center'"
+            />
+            <BuilderButton
+                className="'fa fa-fw fa-align-right'"
+                title.translate="Right"
+                classAction="'o_avatars_end'"
+            />
+        </BuilderButtonGroup>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Extra Content">
+        <BuilderButton
+            className="'flex-basis-50'"
+            title.translate="Show 'More'"
+            action="'avatarsAddContent'"
+            actionParam="{
+                elSelector: '.s_avatars_more',
+                elView: 'website.s_avatars.more',
+                elParent: '.s_avatars_rules_collector',
+            }"
+        >
+            More
+        </BuilderButton>
+        <BuilderButton
+            id="'avatars_has_label_opt'"
+            className="'flex-basis-50'"
+            title.translate="Text"
+            action="'avatarsAddContent'"
+            actionParam="{
+                elSelector: ['.s_avatars_label'],
+                elView: ['website.s_avatars.label'],
+            }"
+        >
+            Label
+        </BuilderButton>
+    </BuilderRow>
+
+    <BuilderRow label.translate="Direction" t-if="isActiveItem('avatars_has_label_opt')">
+        <BuilderButtonGroup>
+            <BuilderButton
+                className="'fa fa-fw fa-long-arrow-right'"
+                title.translate="Row"
+                classAction="'flex-row'"
+            />
+            <BuilderButton
+                className="'fa fa-fw fa-long-arrow-down'"
+                title.translate="Column"
+                classAction="'flex-column'"
+            />
+        </BuilderButtonGroup>
+    </BuilderRow>
+
+    <BuilderContext applyTo="'.s_avatars_rules_collector'">
+        <BorderConfigurator withRoundCorner="false" label.translate="Border"/>
+        <BuilderRow label.translate="Roundness" tooltip.translate="Applies to all avatars and count">
+            <BuilderRange
+                action="'setClassRange'"
+                actionParam="['rounded-0','rounded-1','rounded-2','rounded-3','rounded-4','rounded-5', 'rounded-circle']"
+                max="6"
+                displayRangeValue="false"
+            />
+        </BuilderRow>
+    </BuilderContext>
+</t>
```

Comment:
- I find the "and count" confusing, I think that "Applies to all avatars" is enough to understand, since the "+10" looks like one :slightly_smiling_face: 
- Remove the `displayRangeValue="false"` (= default value) 😉